test/e2e/utils_test.go

```diff
@@ -320,3 +320,28 @@ func getKubernetesLogs(t *testing.T, env environments.Environment, namespace, na
 	}
 	return string(out), nil
 }
+
+// httpGetResponseContains returns true if the response of GETting the URL contains specified substring.
+func httpGetResponseContains(t *testing.T, url string, client *http.Client, substring string) bool {
+	req, err := http.NewRequest("GET", url, nil)
+	if err != nil {
+		t.Log("failed to create request:",err)
```

Comment:
```suggestion
		t.Logf("failed to create request: %v", err)
```

Reply:
ditto elsewhere

Reply:
What is the difference of the two outputs?

Reply:
I'm not sure if there is one but it's IMHO a touch more readable.

Reply:
fixed. `Logf` is indeed more readable.